x-pack/plugins/apm/public/components/alerting/helper.ts

```diff
@@ -15,12 +14,30 @@ export interface AlertMetadata {
   end?: string;
 }
 
-export function getAbsoluteTimeRange(windowSize: number, windowUnit: string) {
-  const now = new Date().toISOString();
+export enum TIME_UNITS {
+  SECOND = 's',
+  MINUTE = 'm',
+  HOUR = 'h',
+  DAY = 'd',
+}
```

Comment:
Doesn't look like you use this at runtime. In that case you can simplify by using a type instead of enum
```suggestion
export type TimeUnits = 's' | 'm' | 'h' | 'd';
```